README.md

```diff
@@ -15,6 +15,9 @@ Don't worry, it only requires some basic knowledge about using the terminal and
 > **Note**
 > Everywhere in this repository, make sure to replace `ublue-os/startingpoint` with the details of your own repository. Unless you used [`create-ublue-image`](https://github.com/EinoHR/create-ublue-image), in which case the previous repo identifier should already be your repo's details.
 
+> **Warning**
+> To start, you *must* create a branch called `live` which is exclusively for your customizations. That is the **only** branch the GitHub workflow will deploy to your container registry. Don't make any changes to the original "template" branch. It should remain untouched. By using this branch structure, you ensure a clear separation between your own "published container" branch, your development branches, and the original upstream "template" branch. Periodically sync and fast-forward the upstream "template" branch to the most recent revision. Then, simply rebase your `live` branch onto the updated template to effortlessly incorporate the latest improvements into your own repository, without the need for any messy, manual "merge commits".
```

Comment:
"published container" -> "published image"

It's a _Native Container_ and it's never containerized. I've tried to take on the effort of always calling them images instead of containers to alleviate confusion.

Reply:
@EinoHR Good point, will replace all mentions of container.